src/style.css

```diff
@@ -1,3 +1,59 @@
+@import 'https://fonts.googleapis.com/css?family=Roboto:100,100italic,300,300italic,regular,italic,500,500italic,700,700italic,900,900italic';
+
 body {
-  margin: 0;
+  font-family: Roboto, sans-serif;
+  max-width: 1024px;
+  margin: 0 auto;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-transform: uppercase;
+  font-size: 15px;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav ul {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  height: 15px;
```

Comment:
The 'height' property for '.nav ul' is set to 15px. Ensure this is necessary for your design, as it might not be needed and could restrict the layout if the content changes.